pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -351,11 +359,12 @@ class CollectionUploadViewSet(
     ViewSet for Collection Uploads.
     """
 
-    authentication_classes = []
-    permission_classes = []
```

Comment:
This enables access control by virtue of the default permission class(es), right?

Reply:
That's correct. These viewsets inherit from the default drf viewsets, which include the default auth and permission classes.